getting-started.md

```diff
@@ -0,0 +1,158 @@
+# Getting Started
+
+## Installation
+The tool works in Mac/Linux/Windows platform and is distributed using NPM.
+```shell
+curl -o- https://raw.githubusercontent.com/nvm-sh/nvm/v0.39.1/install.sh | bash
+source ~/.bashrc
+nvm install 16
+npm i -g @techgreedy/singularity
+```
+If you are using a root user, you may encounter some permission error during installation. If so, use below commands to work around:
```

Comment:
minor: "...If you are _not_ using a root user."

Reply:
no, it happens to root user only.